From 11c1bd59c5e5c285825a48195936985506504be1 Mon Sep 17 00:00:00 2001
From: Csaba Hruska <csaba.hruska@gmail.com>
Date: Mon, 28 Oct 2024 15:20:24 +0100
Subject: [PATCH 2/2] set final stage to stage3

---
 src/UserSettings.hs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/UserSettings.hs b/src/UserSettings.hs
index 5920b57..952a2d9 100644
--- a/src/UserSettings.hs
+++ b/src/UserSettings.hs
@@ -62,4 +62,4 @@ successColour = mkSuccessColour (Dull Green)
 -- 'Stage1' compiler. Setting it to 'Stage3' will build the 'Stage3'
 -- compiler. Setting it to 'Stage0' will mean nothing gets built at all.
 finalStage :: Stage
-finalStage = Stage2
+finalStage = Stage3
-- 
2.43.0

